### DISCUSSION TITLE
Make it compatible with bootstrap 4 on v3

### DISCUSSION THREAD

**Discussion by yepzy** [Okipa/laravel-brickables#21] [Ideas]
Hi,

As your other package https://github.com/Okipa/laravel-table it will be helpful to have the possibility to still work with bootstrap 4 on the v3.

By adding it in the config views, then load the view for the bootstrap 4 or 5

**Comment by Okipa** [comment]
Hi @yepzy it will come with v4 with a bunch of new features and improvements.
